File: python/exercise11/currency_conversion.py
euros = int(input("How many euros are you exchanging? "))
exchange_rate = float(input("What is the exchange rate? "))

to_dollars = (float(euros) * exchange_rate) / 100

print(
    f"{euros} euros at an exchange rate of {exchange_rate} is\n{to_dollars:.2f} U.S. dollars."
)

File: javascript/exercise11/currency-conversion.js
import { input } from "pynput"

const euros = Number(await input("How many euros are you exchanging? "))
const exchangeRate = Number(await input("What is the exchange rate? "))

const toDollars = (euros.toFixed(2) * exchangeRate) / 100
console.log(`${euros} euros at an exchange rate of ${exchangeRate} is\n${toDollars.toFixed(2)} U.S. dollars.`)
